pkg/ccl/backupccl/restore_old_versions_test.go

```diff
@@ -445,3 +452,30 @@ func fullClusterRestoreSystemExternalConnectionsWithoutIDs(exportDir string) fun
 		})
 	}
 }
+
+func fullClusterRestoreWithTenants(exportDir string) func(t *testing.T) {
+	return func(t *testing.T) {
+		tmpDir, tempDirCleanupFn := testutils.TempDir(t)
+		defer tempDirCleanupFn()
+
+		_, sqlDB, cleanup := backupRestoreTestSetupEmpty(t, singleNode, tmpDir,
+			InitManualReplication, base.TestClusterArgs{
+				ServerArgs: base.TestServerArgs{
+					Knobs: base.TestingKnobs{
+						JobsTestingKnobs: jobs.NewTestingKnobsWithShortIntervals(),
+					},
+				}})
+		defer cleanup()
+		err := os.Symlink(exportDir, filepath.Join(tmpDir, "foo"))
+		require.NoError(t, err)
+		sqlDB.CheckQueryResults(t, fmt.Sprintf("SELECT count(*) FROM [SHOW BACKUP LATEST IN '%s'] WHERE object_type = 'TENANT'", localFoo), [][]string{
+			{"2"},
```

Comment:
nit: perhaps it's worth checking that the right tenant names are being shown, and not just the count

Reply:
Because these were created on 22.2, they don't have names in the manifest, only IDs.  We could check the IDs, but in some sense I don't care what they are provided they don't OOM and get restored.